.github/workflows/publish-helm.yaml

```diff
@@ -5,7 +5,7 @@ on:
     types: [published]
 
 permissions:
-  contents: read
+  contents: write
```

Comment:
Probably how to publish the helm we update the release we are going to need to have write permissions. It seems `contents` gives access to `releases`:

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/assigning-permissions-to-jobs